### PR TITLE
Added German translation (and fixed a typo in the README.md)

### PR DESCRIPTION
**Pull request by ToadsworthLP**
I felt like translating another mod, and now it's done!
PS: How much of the 1.8.9 port you promised is currently done?
PPS: If not 1.8.9, will we get it for 1.9.4?

### CODE REVIEW

**Comment by covers1624** [comment]
Cheers for this, ill pull it in when i do the 1.9 update, If you haven't heard ive taken over cb's mods so ive been focusing on those.

**Comment by covers1624** [comment]
PS: 1.9 will probably be a total rewrite and move to CCC as a base.

**Comment by ToadsworthLP** [comment]
Yeah, already noticed you're CB's mods new maintainer. Also noticed the lack of power converting methods in 1.8 and 1.9, sooo... nice to know we'll get one for 1.9 at least! :D

File: src/main/resources/assets/powerconverters/lang/de_DE.lang [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#Common Blocks
+powerconverters.common.bridge.name=Energiebrücke
+powerconverters.common.charger.name=Universelle Ladestation
+
+#Bc Blocks Dead. Kept for no reason at all
+#powerconverters.bc.consumer.name=BC-Verbraucher
+#powerconverters.bc.producer.name=BC-Erzeuger
+
+#IC2
+powerconverters.ic2.lv.consumer.name=IC2 LV-Verbraucher
```
Comment (ToadsworthLP) [inline src/main/resources/assets/powerconverters/lang/de_DE.lang]: ī

Comment (ToadsworthLP) [inline src/main/resources/assets/powerconverters/lang/de_DE.lang]: Noooo Github, I clicked the "Cancel" button.

**Pull request merged**